src/editor/graph/graph_node.cpp

```diff
@@ -379,26 +413,98 @@ Ref<StyleBoxFlat> OrchestratorGraphNode::_make_colored_style(const String& p_exi
             dup->set_bg_color(p_color);
         else
             dup->set_border_color(p_color);
+
+        _apply_corner_radius(dup, p_titlebar);
+
         return dup;
     }
     return sb;
 }
 
-Ref<StyleBoxFlat> OrchestratorGraphNode::_make_selected_style(const String& p_existing_name, bool p_titlebar)
+Ref<StyleBox> OrchestratorGraphNode::_make_selected_style(const String& p_existing_name, bool p_titlebar)
 {
     Ref<StyleBoxFlat> sb = get_theme_stylebox(p_existing_name);
     if (sb.is_valid())
     {
         Ref<StyleBoxFlat> dup = sb->duplicate(true);
-        dup->set_border_color(Color(0.68f, 0.44f, 0.09f));
+        dup->set_border_color(_get_selection_color());
         dup->set_border_width(p_titlebar ? SIDE_TOP : SIDE_BOTTOM, 2);
         dup->set_border_width(SIDE_LEFT, 2);
         dup->set_border_width(SIDE_RIGHT, 2);
+
+        _apply_corner_radius(dup, p_titlebar);
+
         return dup;
     }
     return sb;
 }
 
+Ref<StyleBox> OrchestratorGraphNode::_make_gradient_titlebar_style(const String& p_existing_name, const Color& p_color,
+                                                                   bool p_selected)
+{
+    Ref<Gradient> gradient(memnew(Gradient));
+
+    PackedFloat32Array offsets;
+    offsets.push_back(0);
+    offsets.push_back(1);
+    gradient->set_offsets(offsets);
+
+    PackedColorArray colors = gradient->get_colors();
+    colors.reverse();
+    gradient->set_colors(colors);
+
+    Ref<GradientTexture2D> texture(memnew(GradientTexture2D));
+    texture->set_gradient(gradient);
+    texture->set_width(64);
+    texture->set_height(64);
+    texture->set_fill_to(Vector2(1.1, 0));
+
+    Ref<StyleBoxTexture> titlebar_tex(memnew(StyleBoxTexture));
+    titlebar_tex->set_texture(texture);
+    titlebar_tex->set_modulate(p_color);
```

Comment:
Rather than apply the color on the `modulate` function, we should apply that to override the white component in the above `PackedColorsArray`.  Additionally, the user should be able to supply a secondary Gradient color in the settings that would effectively replace the black. By default, the second color would be `#000000`.